File: android2_lesson01/src/java/java/com/example/ama/android2_lesson01/ui/details/mvp/DetailsNoteView.java
package com.example.ama.android2_lesson01.ui.details.mvp;

import com.example.ama.android2_lesson01.ui.base.NoteView;

/**
 * Interface for work with {@link DetailsNotePresenter}
 */

public interface DetailsNoteView extends NoteView {
    void finishEditing();
}

File: android2_lesson01/src/kotlin/java/com/example/ama/android2_lesson01/ui/details/mvp/DetailsNoteView.kt
package com.example.ama.android2_lesson01.ui.details.mvp

import com.example.ama.android2_lesson01.ui.base.NoteView

/**
 * Interface for work with [DetailsNotePresenter]
 */

interface DetailsNoteView : NoteView {
    fun finishEditing()
}